spec/features/reviews_spec.rb

```diff
@@ -35,6 +35,15 @@
         click_on('Dlaczego solipsyzm?, v. 1')
         expect(page).to have_css(".btn", text:"Edytuj")
       end
+      
+      scenario "testowanie usuwania artykułu z recenzją" do
+        visit '/submissions'
+        fill_in "Tytuł", with: "Arystoteles."
+        click_on("Filtruj")
+        page.find(".btn-danger", :match => :first).click
+        
+        expect(page).to have_css(".has-error")
```

Comment:
To niestety nie zadziała ponieważ nie ma formularza, w którym mogłaby pojawić się ta klasa. Trzeba zmodyfikować metodę `delete` w `submissions_controller`, tak by sprawdzała czy usunięcie zakończyło się powodzeniem. Jeśli się nie zakończy, to powinien być ustawiany `flash[:error]` z informacją, że nie można usunąć zgłoszenia. I to powinno być tutaj testowane.